### DISCUSSION TITLE
Download future videos from subscriptions

### DISCUSSION THREAD

**Discussion by Coayer** [Tzahi12345/YoutubeDL-Material#340] [Q&A] [answered]
I'm trying to set up my subscriptions so that I don't download any videos already uploaded, but download videos posted from now onwards. Is there a way I can accomplish this?

**Comment by Tzahi12345** [comment, marked as answer]
Yes this is very easy. Just uncheck "download all videos" and set it to download videos uploaded in the last 1 day. That way only new videos will be downloaded!

**Reply by Coayer** [reply]
Ah, great. I'll give that a go, thank you!

**Reply by GlassedSilver** [reply]
Going through some older discussions I just pondered about this one for a little bit:

If you ever have scheduled or unscheduled downtime of electricity, internet, your hardware or your software that all serves up your instance of YoutubeDL-Material that lasts for more than the value that you specified, this will become a point of failure possibly if there are any uploads in that time-window.

Best to increase the time-window to something like 7 days, maybe. I think 7 days of outage should be pretty good for starting such a sub, later on you can feel free to increase it even further, because retroactively nothing new will be pulled and you just increase your safety-net. Just food for thought.
